src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -240,36 +215,34 @@ const PathwaysListElement: FC<PathwaysListElementProps> = ({
                 ))}
               </tbody>
             </table>
-            <button
-              {...selectButtonOpts}
-              className={indexStyles.button}
-              disabled={selected}
+            <Button
               onClick={(): void => {
-                const carePlan = createCarePlan(pathway.name, patient);
-
-                setPatientRecords([...patientRecords, carePlan]);
-                client?.create?.(carePlan);
-                callback(evaluatedPathway);
+                if (selected) unassignPathway(pathway.name);
+                else assignPathway(pathway.name);
```

Comment:
Might look cleaner here with a ternary instead of if else
```
               selected ? unassignPathway(pathway.name) : assignPathway(pathway.name);
```